conf.py

```diff
@@ -57,9 +57,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '7.0'
+version = '7.1'
 # The full version, including alpha/beta/rc tags.
-release = '7.0.3'
```

Comment:
Dropping the patch version here because I don't foresee docs ever showing different content across patch versions.

Reply:
What is this flag used for? Do we expect any issues dropping this precision?

Reply:
Read The Docs makes versions from these numbers. Someone sent a PR a few months ago and bumped this field to "6.2.2." Well unknown to me all the new docs content was being rendered in that version even though v6.2 was the default version visitors would see when they went to docs.citusdata.com. So for a while the newest changes appeared to not be reflecting on the docs. I noticed this extra version in the docs selector in the lower left, then marked 6.2.2 as private and restored the version to 6.2.0.

I'm removing the extra digit so that nobody is tempted to change it and accidentally create a new docs version.

Reply:
Sphinx docs distinguish between `version` and `release` as:

___

### version

The major project version, used as the replacement for |version|. For example, for the Python documentation, this may be something like 2.6.

### release

The full project version, used as the replacement for |release| and e.g. in the HTML templates. For example, for the Python documentation, this may be something like 2.6.0rc1.

If you don’t need the separation provided between version and release, just set them both to the same value.